### ISSUE TITLE
[Feedback]: Sudden Paywall Blocking Access to Ongoing Training Data & Status of Early Adopter Accounts

### ISSUE THREAD

**Issue by PeiPeiWow** [wandb/wandb#12477]
### Describe the bug

To the W&B Team and Community,

I am opening this issue to raise a critical concern regarding the recent, unannounced changes to account tiers and data accessibility. This recent shift is significantly impacting user trust and disrupting ongoing work across the community, particularly for early adopters.

There are two major issues that need immediate public clarification and resolution:

1. Sudden Loss of Access to Ongoing Training Data (Critical Block)
Many users are suddenly finding themselves locked out of viewing their currently running training jobs due to the new subscription enforcement.

While the community fully understands a company's need to introduce paid tiers and monetize advanced features, completely severing a user's access to their own active data without a grace period or prior warning is highly anti-consumer. Charging for a service is standard practice; however, effectively paywalling ongoing research and preventing users from even viewing data they are currently generating is an unacceptable user experience. It breaks the fundamental trust developers place in W&B as a reliable experiment tracking tool.

2. The Status of the "Free Forever" Commitment
When W&B first launched, early adopters were assured that their accounts would remain free. The recent forced migration to paid plans appears to directly contradict this initial commitment. Many of us have advocated for, championed, and supported W&B for years based on this mutual trust.

Actionable Requests for the W&B Team:
On behalf of the affected users, we request the following:

Immediate Data Access: Restore read-access to ongoing/existing training runs for all affected users so that active research and compute resources are not wasted.

Official Clarification: Provide a transparent, public update regarding the status of early adopter accounts and why the initial "free forever" promise is no longer being honored.

Fair Transition Policies: Implement reasonable grace periods for any future pricing or tier changes to prevent sudden lockouts.

We have deeply appreciated the platform W&B has built over the years, but these recent actions severely damage user trust. We look forward to an official response, a clear explanation, and a constructive resolution for the community.

<img width="2472" height="1136" alt="Image" src="https://github.com/user-attachments/assets/00f06f35-ebca-4b10-a82d-ff831a31a66b" />

**Comment by PeiPeiWow** [comment]
Just want to share my personal tragedy from last night to highlight how wildly abrupt this is... 😂😭

Last Night (The Dream):
I proudly executed my training script. The GPUs were humming, the loss was dropping beautifully. I went to bed with a peaceful, fatherly smile, dreaming of perfectly converging loss curves and state-of-the-art results. 🛌✨📉

---

This Morning (The Nightmare):
I jumped out of bed like a kid on Christmas morning, brewed my coffee ☕, and eagerly opened my W&B dashboard to check on the "pills" I'd been alchemy-ing all night.
And what do I get? BAM! A giant Paywall slammed directly into my face. 🚪💥

I was absolutely shocked, and then I literally wanted to cry! 😭😭😭
I can't even see the models I left baking in the oven all night! Did they converge? Did the gradients explode? Did the models achieve AGI during the night and decide to lock me out themselves?! I HAVE NO IDEA!!!

<img width="320" height="180" alt="Image" src="https://github.com/user-attachments/assets/11405d13-feda-4b51-9a9f-969a55f40f4e" />

I totally understand that servers cost money and a business needs to be profitable. But locking us out of currently running training data with zero warning feels like changing the locks on my house while my dog is still inside! 🐶

Please, W&B team... at least let me look through the window to see if my overnight run survived before handing me the bill! 🙏💸

**Comment by PeiPeiWow** [comment]
Hi @cgvarghese,

Thank you for looking into this!

For privacy reasons, I would prefer not to share my account details (email or org name) publicly here on GitHub. However, I am currently in contact with your support team (Paulo) via email regarding this exact issue. I have just provided my account details and the context of your comment in my email reply to him.

Could you please sync with Paulo internally to check my account status? I really appreciate your help in getting this resolved!

**Comment by PeiPeiWow** [comment]
You previously stated that after migrating from personal entities to organizations, the commitment to early users would remain unchanged, including our corresponding storage limits and associated rights.

However, the only solution your support team is giving me now is to switch my account to the "Free Plan". The problem is, if I switch to this Free Plan, all my actual benefits change—for example, the storage drops to only 5GB. This completely contradicts the permanent rights you promised to early W&B users.

I have already submitted an issue previously and communicated with your support team multiple times, yet the problem remains unresolved.

So my question now is: Are the permanent rights you promised to early users just gone?

<img width="1569" height="286" alt="Image" src="https://github.com/user-attachments/assets/f39df478-4072-459d-91d1-5f10f9c52d45" />
